### DISCUSSION TITLE
Feature Request: kiosk / cast screen

### DISCUSSION THREAD

**Discussion by Wouter0100** [mistweaverco/bananas#61] [Ideas]
It would be really awesome to be able to have a Raspberry PI (or any computer) attached to a screen, after which you're able to share your screen to that Pi.

Could maybe be combined with the Web UI feature?

**Comment by gorillamoe** [comment]
Not against this, but would like to have this in a separate package then.

You should be able to stream from usual bananas 🍌 app to the kiosk client, but that client should only have to run a minimal version of bananas, maybe just a cli version.

If it gets enough up votes, we start implementing it.

**Reply by Wouter0100** [reply]
Yes, awesome!
